src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx

```diff
@@ -0,0 +1,191 @@
+import { Trans } from '@lingui/macro'
+import React, { ErrorInfo } from 'react'
+import store, { AppState } from 'state/index'
+import { ExternalLink, TYPE } from 'theme/index'
+import { AutoColumn } from 'components/Column'
+import styled from 'styled-components/macro'
+import ReactGA from 'react-ga'
+import { getUserAgent } from 'utils/getUserAgent'
+import { AutoRow } from 'components/Row'
+
+const FallbackWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  z-index: 1;
+`
+
+const BodyWrapper = styled.div<{ margin?: string }>`
+  padding: 1rem;
```

Comment:
Could use px's to match other values.

Reply:
It's a mod, i didn't touch this part. We can probably redo this component completly. For now just hotfix